bindings/python/examples/first_frame/first_frame.py

```diff
@@ -112,6 +112,11 @@ def callbackFunction(callbackStatus):
 # currentIniParams["ab_thresh_min"] = 4
 # camera1.setIniParams(currentIniParams)
 
+# Example of configuring the Dynamic Mode Switching
+# The expected sequence of frame is: mode2, mode2, mode2, mode3, mode2, ...
+# camera1.adsd3500setEnableDynamicModeSwitching(True)
+# camera1.adsds3500setDynamicModeSwitchingSequence([(2, 3,), (3, 1)])
```

Comment:
I think there is an extra comma after the first 3 in the sequence

Reply:
yep, that shouldn't be there. Thanks!